synapse/storage/databases/main/events_worker.py

```diff
@@ -835,7 +835,7 @@ def _maybe_redact_event_row(self, original_ev, redactions, event_map):
                  which we can look up the redaaction events. Map from event id to event.
 
         Returns:
-            Deferred[EventBase|None]: if the event should be redacted, a pruned
+            EventBase|None: if the event should be redacted, a pruned
```

Comment:
If you're fixing the type signature may as well add type hints? The ones in the docstring look correct

Reply:
I must have missed this one, sorry!